int32 workerID
---
bool isActive
bool isComplete
int32 workerID
int32 zone
int32 priority

